database/migrations/2018_08_03_103854_create_reviews_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateReviewsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('reviews', function (Blueprint $table) {
+            $table->increments('id');
+            $table->text('description');
+            $table->unsignedInteger('user_id');
+            $table->unsignedInteger('place_id');
```

Comment:
Make a foreign key on `place_id`, it doesn't matter that a table doesn't exist yet